edxlearndot/management/commands/update_learndot_enrolments.py

```diff
@@ -17,7 +17,7 @@
 from lms.djangoapps.courseware.courses import get_course
 from lms.djangoapps.grades.config import should_persist_grades
 from lms.djangoapps.grades.course_grade_factory import CourseGradeFactory
-from student.models import CourseEnrollment
+from common.djangoapps.student.models import CourseEnrollment
```

Comment:
@eLRuLL  FYI I added this change to avoid this error, which will cause things to fail in Koa:

> import_shims.warn.DeprecatedEdxPlatformImportError: Importing student instead of common.djangoapps.student is deprecated